yb-voyager/cmd/exportData.go

```diff
@@ -507,10 +507,14 @@ func clearDataIsExported() {
 }
 
 func updateSourceDBConfInMSR() {
+	if exporterRole != SOURCE_DB_EXPORTER_ROLE {
+		return
+	}
 	metaDB.UpdateMigrationStatusRecord(func(record *metadb.MigrationStatusRecord) {
 		if record.SourceDBConf == nil {
 			record.SourceDBConf = source.Clone()
 			record.SourceDBConf.Password = ""
+			record.SourceDBConf.Uri = ""
```

Comment:
can we also make this change of Uri on the import side for target-db/ff-db as in case of ff-db URI is getting set in msr.